src/fable/Fable.Compiler/Replacements.fs

```diff
@@ -1121,6 +1121,8 @@ module private AstPass =
         | "find" when Option.isSome c ->
             let defaultValue = defaultof i.calleeTypeArgs.Head 
             ccall "Seq" "tryFind" [args.Head;c.Value;defaultValue] |> Some
+        | "findAll" when Option.isSome c ->
+            ccall "Seq" "filter" [args.Head;c.Value] |> Some
```

Comment:
`FindAll` produces a new ResizeArray so you need make one from the result of `Seq.filter` (see [this](https://github.com/fable-compiler/Fable/blob/6f44e97c1268b2707e73aef22009f237c3a2ee1d/src/fable/Fable.Compiler/Replacements.fs#L1217-L1222)). It should be fixed by doing:

```fsharp
ccall "Seq" "filter" [args.Head;c.Value] |> toArray com i |> Some
```